src/main/java/com/fcgl/madrid/search/dataModel/Customer.java

```diff
@@ -0,0 +1,27 @@
+package com.fcgl.madrid.search.dataModel;
+import org.springframework.data.annotation.Id;
+
+public class Customer {
```

Comment:
This class creates a collection in mongodb. Collection name `customer` with attributes `Id, firstName and latName`